roles/openshift_common/vars/main.yml

```diff
@@ -6,4 +6,4 @@
 # interfaces)
 os_firewall_use_firewalld: False
 
-openshift_data_dir: /var/lib/openshift
+openshift_data_dir: /var/lib/{{ product_type }}
```

Comment:
I thought I heard somewhere that this will be staying the same now between products...  @brenton please correct me if I'm wrong here.